linera-service/src/storage.rs

```diff
@@ -123,19 +125,28 @@ impl FromStr for StorageConfig {
         if let Some(s) = input.strip_prefix(SCYLLA_DB) {
             let mut uri: Option<String> = None;
             let mut table_name: Option<String> = None;
+            let parse_error: &'static str = "Correct format is tcp://db_hostname:port.";
```

Comment:
It seems not to be like that and instead to be `tcp:db_hostname:port`.

Reply:
Good point! Thanks :)